qa/tasks/cephfs/test_volumes.py

```diff
@@ -6518,7 +6518,11 @@ def test_subvolume_snapshot_clone_quota_exceeded(self):
         self._fs_cmd("subvolume", "create", self.volname, subvolume,"--mode=777", "--size", str(osize))
 
         # do IO, write 50 files of 1MB each to exceed quota. This mostly succeeds as quota enforcement takes time.
-        self._do_subvolume_io(subvolume, number_of_files=50)
+        try:
+            self._do_subvolume_io(subvolume, number_of_files=50)
+        except CommandFailedError:
+            # command failed due to Disk quota exceeded as quota enforcement takes some.
```

Comment:
we'd need to check for errno being EDQUOT.

Reply:
@nmshelke mentioned about the interface not returning appropriate error codes (returns 1 on failures).

Let's leave this as it is. Looks fine!

Reply:
@nmshelke Doesn't the following code work?

```
 except CommandFailedError as e:
     if e.exitstatus == EDQUOT:
         pass

```